walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -37,6 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Convert the server stats to JSON
     ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
+    ServerStats serverStats2 = new ServerStats(startTime, curretn)
```

Comment:
`curretn` looks like a misspelling of `currentTime`

Reply:
I think this line is actually not needed and I forgot why I added this line in the example file. After removing the line, the whole servlet file still works great and maybe I added this line by mistake. I removed this line in the newest commit.